vignettes/installation.Rmd

```diff
@@ -33,8 +33,15 @@ remotes::install_cran('azuremlsdk', repos = 'https://cloud.r-project.org/', INST
 ```
 
 ## Install the Azure ML Python SDK
-Lastly, use the **azuremlsdk** R library to install the Python SDK. By default, `azuremlsdk::install_azureml()` will install the [latest version of the Python SDK](https://pypi.org/project/azureml-sdk/) in a conda environment called `r-azureml` if reticulate < 1.14 or `r-reticulate` if reticulate ≥ 1.14.
-``` {r install_pythonsdk, eval=FALSE}
+Lastly, use the **azuremlsdk** R library to install the [latest version of the Python SDK](https://pypi.org/project/azureml-sdk/).
+
+If using an Azure Machine Learning Compute Instance or a CRAN installation with `reticulate >= 1.14`, use the `azuremlsdk::install_azureml()` function with the conda environment set to `r-reticulate`.
+``` {r install_pythonsdk1, eval=FALSE}
+azuremlsdk::install_azureml(envname = 'r-reticulate')
```

Comment:
should we change the `install_azureml()` parameter back to 'envname' instead of 'custom_envname'

Reply:
Good point - I was having a similar thought to prevent confusion. I'll add it to this PR/